packages/blade/src/components/Button/IconButton/IconButton.tsx

```diff
@@ -108,6 +110,7 @@ const _IconButton: React.ForwardRefRenderFunction<BladeElementRef, IconButtonPro
       onTouchEnd={onTouchEnd}
       onTouchStart={onTouchStart}
       {...makeAnalyticsAttribute(rest)}
+      {...rest}
```

Comment:
```suggestion
      {...getStyledProps(rest)}
```

did you mean this? or is it handled internally?

Reply:
oh got it. You're using useStyledProps. Makes sense. Ignore my last comment